### PR TITLE
Added mwcs templates

### PR DESCRIPTION
**Pull request by rbawden**

### CODE REVIEW

**Review by VictorSanh — CHANGES_REQUESTED** [review]
lgtm thanks @rbawden !

Small comment (please above)

File: templates/mwsc/templates.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    jinja: '{{ question|trim(''?'') }} in the sentence "{{ sentence|trim(''.'') }}"?
+      ||| {{ answer }}'
+    name: in-the-sentence-question-first
+    reference: ''
+  8d4f3463-d64b-43be-b0ed-2455cb99e017: !Template
+    id: 8d4f3463-d64b-43be-b0ed-2455cb99e017
+    jinja: 'If I were to say "{{sentence}}" and then ask you "{{ question }}", what
+      do you think is the correct answer out of
+
+      {% for option in options %}"{{ option }}"{% if loop.index == options|length
```
[the remark this answers is not in the document]
Comment (rbawden) [reply templates/mwsc/templates.yaml:14]: Ok! I agree - there are only 2 options in this dataset (I was doing a more generic template that could work for other datasets too). So I'm simplifying this. I assume this is also applicable to the other templates I've done here too where a join would also work?

[the remark this answers is not in the document]
Comment (VictorSanh) [reply templates/mwsc/templates.yaml]: awesome thanks!

**Comment by VictorSanh** [comment]
merging now, please update the spreadhseet @rbawden !

**Review by VictorSanh — APPROVED** [review]

**Pull request merged**